packages/material-ui/src/Typography/Typography.js

```diff
@@ -7,7 +7,7 @@ import capitalize from '../utils/capitalize';
 import typographyClasses, { getTypographyUtilityClass } from './typographyClasses';
 
 const getTextColor = (color, palette) => {
-  if (color === 'textPrimary' || color === 'textSecondary') {
+  if (color.startsWith('text')) {
```

Comment:
`startsWith` is not supported by IE11 https://caniuse.com/?search=startsWith Would recommend just checking if `color.indexOf('text') === 0`